scipy/optimize/zeros.py

```diff
@@ -4,21 +4,29 @@
 from collections import namedtuple
 from . import _zeros
 import numpy as np
+from numpy import (finfo, sign, isfinite, abs, asarray, nan, imag, isclose,
+                   frexp, zeros as npzeros, sum as npsum, diff as npdiff,
+                   array as nparray)
```

Comment:
nitpick: the `zeros as npzeros` etc looks quite odd. why add any of these imports instead of just using the `np.` prefix?

Reply:
`abs` and `sum` shadow builtins too, so those also need to be avoided as `from numpy` imports

Reply:
Importing `np.zeros` as `npzeros` was required to avoid overriding `zeros`, and yes it looks a little odd.  The idiom `from numpy import XXX, YYY` appears frequently in SciPy .py/.pyx/.pxd files, possibly for faster symbol lookup, or reduced typing.  In any case, I've replaced all these with the full `np.XXX`.